plugins/woocommerce/includes/class-wc-structured-data.php

```diff
@@ -231,12 +231,15 @@ public function generate_product_data( $product = null ) {
 				if ( $lowest === $highest ) {
 					$markup_offer = array(
 						'@type'              => 'Offer',
-						'price'              => wc_format_decimal( $lowest, wc_get_price_decimals() ),
-						'priceValidUntil'    => $price_valid_until,
 						'priceSpecification' => array(
-							'price'                 => wc_format_decimal( $lowest, wc_get_price_decimals() ),
-							'priceCurrency'         => $currency,
-							'valueAddedTaxIncluded' => wc_prices_include_tax() ? 'true' : 'false',
+							array(
+								'@type'                 => 'UnitPriceSpecification',
+								'priceType'             => 'https://schema.org/ListPrice',
+								'price'                 => wc_format_decimal( $lowest, wc_get_price_decimals() ),
+								'priceCurrency'         => $currency,
+								'valueAddedTaxIncluded' => wc_prices_include_tax() ? 'true' : 'false',
```

Comment:
We could simplify this since we touch it anyway.
```suggestion
								'valueAddedTaxIncluded' => wc_prices_include_tax(),
```
If you agree, please also update the other occurrences. Thanks!